bottlerocket/agents/src/sonobuoy.rs

```diff
@@ -181,6 +208,9 @@ pub async fn wait_for_sonobuoy_results(
         let run_status: serde_json::Value = match serde_json::from_str(&stdout) {
             Ok(run_status) => run_status,
             Err(err) => {
+                results.other_info = Some(format!("Status failed '{}'", retries));
+                info_client.send_test_update(results.clone()).await?;
```

Comment:
Same as previous comments, we are taking on additional risk of a transient network blip ruining the party.